app/common/src/cards/__tests__/SimpleCard.test.js

```diff
@@ -5,14 +5,16 @@ import renderer from 'react-test-renderer';
 
 import SimpleCard from '../SimpleCard';
 
+function onPressVoidCallback() {}
```

Comment:
I will try it again. I pretty sure that in some cases the array function instead of normal function caused the same problem I am trying avoid here. Let me check later... :)

Reply:
Ok, in this case, it's not a problem and in other situations it's avoidable. But let's agree on one thing: we should not write these things in code reviews because it should be enforced by eslint rules. Instead, let's focus on real issues with code design and similar. If we don't like some code style (and I agree it's weird, similar to class methods `likeThis` or `_likeThis`) we should send PR with new eslint rule. Does it make sense for you? (to focus on important things in code reviews and enforcing good habits by eslint)

Reply:
of course, I agree, I expect the usual scenario: discussion in PR => (agreement on setting new rule => new PR) || not important 🙂